src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
I borrow this from `WebApplication::action_command_setSuperSeeding()` and `WebApplication::action_command_setForceStart()`, which use `"true"` as well, i.e. `bool value = request().posts["value"] == "true";`.